packages/flutter_rx_bloc/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [6.2.0]
+* Update `rx_bloc` to use version `5.1.0`
```

Comment:
When upgrading dev dependencies we do not need to bump the version of the package